gitops/argo-apps/sealed-secrets.yaml

```diff
@@ -11,4 +11,4 @@ spec:
   source:
     chart: sealed-secrets
     repoURL: https://bitnami-labs.github.io/sealed-secrets
-    targetRevision: 1.16.1
```

Comment:
Would you also please update the README.md file so that the `sealed-secrets` entry points to the latest version `0.18.5` (that's the actual version, not the helm chart version)?